float32 tsl_full
float32 lps_temp
float32 lps_pressure
float32 sht_humidity
float32 sht_temp
